docs/timeseries.md

```diff
@@ -129,97 +133,69 @@ The SWOT data fields to return in the request.
 
 This is specified in the form of a comma separated list (without any spaces): `fields=reach_id,time_str,wse,slope`
 
+**NOTE: Units are always returned for fields that have corresponding units stored in Hydrocron.**
+
 Hydrocron includes additional fields beyond the source data shapefile attributes, including units fields on measurements, cycle and pass information, and SWORD and collection versions. The complete list of fields that are available through Hydrocron are below:
```

Comment:
Edit this line to say "The complete list of input fields...." or something to that effect. I wonder if moving the note on units makes more sense embedded into this line too? So it reads:
Hydrocron includes additional fields beyond the source data shapefile attributes, including units fields on measurements, cycle and pass information, and SWORD and collection versions. **NOTE: Units are always returned for fields that have corresponding units stored in Hydrocron, they do not need to be requested.** The complete list of input fields that are available through Hydrocron are below:

Reply:
Added!!